e2e/tests/interchain_accounts/base_test.go

```diff
@@ -33,18 +31,6 @@ type InterchainAccountsTestSuite struct {
 	testsuite.E2ETestSuite
 }
 
-// getICAVersion returns the version which should be used in the MsgRegisterAccount broadcast from the
-// controller chain.
-func getICAVersion(chainAVersion, chainBVersion string) string {
```

Comment:
I moved this functions to `intertx_test.go` since that's the only file where it is used now.